src/Uno.UWP/Data/Pdf/PdfDocument.Android.cs

```diff
@@ -0,0 +1,112 @@
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Android.Graphics.Pdf;
+using Android.OS;
+using Windows.Foundation;
+
+namespace Windows.Data.Pdf;
+
+public partial class PdfDocument : IDisposable
+{
+	private readonly PdfRenderer _pdfRenderer;
+
+	private PdfDocument(PdfRenderer pdfRenderer)
+	{
+		this._pdfRenderer = pdfRenderer ?? throw new ArgumentNullException(nameof(pdfRenderer));
+	}
+
+	public bool IsPasswordProtected => false;
+
+	public uint PageCount => (uint)_pdfRenderer.PageCount;
+
+	public PdfPage GetPage(uint pageIndex)
+	{
+		if (pageIndex >= _pdfRenderer.PageCount)
+		{
+			throw new ArgumentOutOfRangeException(nameof(pageIndex), $"In this document the page index cannot be {pageIndex}. Page count is {_pdfRenderer.PageCount}");
+		}
+
+		var pdfPage = _pdfRenderer.OpenPage((int)pageIndex);
+		return new PdfPage(pdfPage);
+	}
+
+	public static IAsyncOperation<PdfDocument> LoadFromFileAsync(Storage.IStorageFile file)
+	{
+		return LoadFromFileAsync(file, null);
+	}
+
+	public static IAsyncOperation<PdfDocument> LoadFromFileAsync(Storage.IStorageFile file, string password)
+	{
+		if (!string.IsNullOrWhiteSpace(password))
+		{
+			// password protected PDF files are not supported by PdfRenderer in Android
+			throw new NotImplementedException("The member IAsyncOperation<PdfDocument> PdfDocument.LoadFromFileAsync(IStorageFile file, string password) is not implemented in Uno.");
+		}
+
+		if (file is null)
+		{
+			throw new ArgumentNullException(nameof(file));
+		}
+
+		var localpath = file.Path;
+		var fileDescriptor = localpath.StartsWith('/')
+			? ParcelFileDescriptor.Open(new Java.IO.File(localpath), ParcelFileMode.ReadOnly)
+			: Android.App.Application.Context.ContentResolver.OpenFileDescriptor(Android.Net.Uri.Parse(localpath), "r");
+		var pdfRenderer = new PdfRenderer(fileDescriptor);
+		var pdfDocument = new PdfDocument(pdfRenderer);
+
+		return Task.FromResult(pdfDocument).AsAsyncOperation();
+	}
+
+	public static IAsyncOperation<PdfDocument> LoadFromStreamAsync(Storage.Streams.IRandomAccessStream inputStream)
+	{
+		return LoadFromStreamAsync(inputStream, default);
+	}
+
+	public static IAsyncOperation<PdfDocument> LoadFromStreamAsync(Storage.Streams.IRandomAccessStream inputStream, string password)
+	{
+		if (!string.IsNullOrWhiteSpace(password))
```

Comment:
```suggestion
		if (password is not null)
```

Reply:
Thank you for report. The "password is not null" is equivalent to "ReferenceEquals(null, password)". UWP considering empty string as no password set. I fix the issue replacing IsNullOrWhiteSpace with IsNullOrEmpty.